<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCA4CDDialogBase</class>
 <widget class="QDialog" name="PCA4CDDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCA4CD - Compute the Principal Components</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QgsMessageBar" name="MsgBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="QGBox_InputData">
        <property name="title">
         <string>Input Rasters Data</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Select one or two (as reference and target) input layer, each band in A (and B) is a variable for the PCA:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignJustify|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_0" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="EnableInputData_A">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>A:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QgsMapLayerComboBox" name="QCBox_InputData_A">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
                 </property>
                 <property name="frame">
                  <bool>true</bool>
                 </property>
                 <property name="allowEmptyLayer">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="QPBtn_browseData_A">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Browse</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_1" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="EnableInputData_B">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>B:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QgsMapLayerComboBox" name="QCBox_InputData_B">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
                 </property>
                 <property name="frame">
                  <bool>true</bool>
                 </property>
                 <property name="allowEmptyLayer">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="QPBtn_browseData_B">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Browse</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="QGBox_PrincipalComponents">
        <property name="title">
         <string>Principal Components</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Number of components:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="QCBox_nComponents"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Estimator matrix:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="QCBox_EstimatorMatrix">
              <item>
               <property name="text">
                <string>Correlation</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Covariance</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Process Se&amp;ttings</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QWidget" name="group_ProcessSettings" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="QPBtn_runPCA">
           <property name="text">
            <string>Compute Principal Components</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_plugin_info" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_25">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="QPBtn_PluginDocs">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>WhatsThisCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Open the PCA4CD documentation</string>
           </property>
           <property name="text">
            <string>Docs</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/plugins/pca4cd/icons/about.svg</normaloff>:/plugins/pca4cd/icons/about.svg</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="QPBtn_PluginClearReload">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Clear all and reload the PCA4CD plugin.</string>
           </property>
           <property name="text">
            <string>Clear/Reload</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/plugins/pca4cd/icons/reload.svg</normaloff>:/plugins/pca4cd/icons/reload.svg</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="QPBtn_PluginInfo">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>WhatsThisCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>About</string>
           </property>
           <property name="text">
            <string>version</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/plugins/pca4cd/icons/pca4cd.svg</normaloff>:/plugins/pca4cd/icons/pca4cd.svg</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMessageBar</class>
   <extends>QFrame</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>groupBox_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>group_ProcessSettings</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EnableInputData_B</sender>
   <signal>clicked(bool)</signal>
   <receiver>widget_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
